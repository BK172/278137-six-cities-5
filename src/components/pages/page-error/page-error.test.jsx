import React from "react";
import renderer from "react-test-renderer";
import PageError from "./page-error";

describe(`PageError component test`, () => {
  it(`Should render PageError correctly`, () => {
    const tree = renderer
      .create(
          <PageError />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
