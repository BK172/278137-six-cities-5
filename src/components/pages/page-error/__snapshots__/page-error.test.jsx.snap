// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PageError render correctly 1`] = `
<div
  className="container"
>
  <h1>
    Network error...
  </h1>
</div>
`;
